mk/dependency-list.sh

```diff
@@ -106,7 +106,7 @@ function get_spk_dependencies ()
    local cumulated_dependencies=$(get_dependencies "${dependencies}")
    while [ "${cumulated_dependencies}" != "${dependencies}" ]; do
       dependencies=${cumulated_dependencies}
-      cumulated_dependencies=$(get_dependencies "${dependencies}")
+      cumulated_dependencies=$(get_dependencies "${dependencies}" | tr ' ' '\n' | sort -u | tr '\n' ' ')
```

Comment:
Naive question: isn't this already done by the `get_dependencies` function ?

Reply:
yes for recursive calls it is, but not im `get_file_dependencies` when python and or ffmpeg dependencies are added.

(maybe the _unique sort filter_ should be added at line 106 too)

Reply:
@smaarn your question was justified. 
This is now updated, and the test for python and ffmpeg packages is optimized to evaluate on top level (spk folder) only.